purchases/src/main/kotlin/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -265,6 +270,14 @@ internal class HTTPClient(
             debugLog { NetworkStrings.API_REQUEST_STARTED.format(connection.requestMethod, path) }
             responseCode = connection.responseCode
             payload = inputStream?.let { readFully(it) }
+            if (ENABLE_EXTRA_REQUEST_LOGGING) {
+                verboseLog { "Request extra logging. Response code: $responseCode. Body: $payload" }
```

Comment:
What do you think of this? 

```suggestion
                verboseLog { "HTTP response:\\n  status code: $responseCode \\n  body: $payload" }
```